app-utils/bili-sync/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=bili-sync
+PKGSEC=utils
+PKGDEP="gcc-runtime"
+PKGDES="Bilibili synchronization tool powered by Rust & Tokio"
```

Comment:
Drop "powered by Rust & Tokio" as it's really useless information for the end-user. Also what does it mean that it's a "synchronization tool"? What does it actually do?